packages/riipen-ui/src/components/Input.jsx

```diff
@@ -11,16 +11,17 @@ import Typography from "./Typography";
 const Input = ({
   classes,
   disabled,
+  end,
```

Comment:
I wonder if we can be slightly more descriptive with these? Maybe just straight copy material-ui with endAdornment and startAdornment?